apps/boilerplate-front-end/src/components/header/components/OktaAuth.jsx

```diff
@@ -0,0 +1,272 @@
+import { OktaLogo } from '@/assets/svg/SvgIndex'
+import { useOktaAuth } from '@okta/okta-react'
+import { useEffect, useState } from 'react'
+
+const OktaAuth = () => {
+  const { authState, oktaAuth } = useOktaAuth()
+  const handleLogin = () => oktaAuth.signInWithRedirect()
+  // To use for a Logout function
+  // const handleLogout = () => oktaAuth.signOut()
+  const [userEmail, setUserEmail] = useState(null)
+  const [imgLink, setLinkImg] = useState(null)
+  // Get info from okta on user
+  const user = oktaAuth.token
+    .getUserInfo()
+    .then(function (user) {
+      // user has details about the user
+      setUserEmail(user.email)
+    })
+    .catch(function (err) {
+      console.log(err)
```

Comment:
shouldn't you raise the error to the UI so the user can report it to us?

Reply:
Good point 👍